impress-org/givewp
# Fix: adjust Campaign overview tab styles

Resolves [GIVE-2029]

## Description
This PR includes the following UI changes to the Campaign overview tabs:

- Update padding on each tab.
- Included line-height.
- Expand hover border to spread width of tab.
- Set a fixed width on tabs to maintain size, while font-weight is changed on hover.
-  Specify `Forms` tab container padding to keep alignment with title. `Forms` page content spans full width and requires  specific styling.
- Use font-weight 500 on hover.
- Use `givewp-neutral` font colors for active/hover state.
- Update spacing & alignment in breadcrumb.
- Update gap in title, badge, campaign buttons container.
- Additional header font changes.
- Header element vertical spacing.

## Affects
- Campaign overview tabs

## Visuals
https://www.loom.com/share/cb3e7695f0dc46a4a4629f94c4559ccf?sid=2e31436f-4aae-4e95-b8e4-66964e1001a2

## Testing Instructions
- View the Campaign overview page.
- Hover over each tab, ensure that the tab widths do not change size due to the font weight becoming bold.
- While hovering the first tab, verify the bottom border is aligned with the title and page content.
- Select each tab and verify the page content is displayed properly and aligned.
- Ensure selecting the forms tab does not have any unintended sideeffects.

## Pre-review Checklist



[GIVE-2029]: https://stellarwp.atlassian.net/browse/GIVE-2029?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
